pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/buffer/TransactionBufferCloseTest.java

```diff
@@ -61,7 +61,7 @@ public Object[][] isPartition() {
         };
     }
 
-    @Test(timeOut = 10_000, dataProvider = "isPartition")
+    @Test(dataProvider = "isPartition")
```

Comment:
Will it get stuck forever? Perhaps setting a timeout of 30 seconds would be safer

Reply:
ok

Reply:
`checkSnapshotPublisherCount` will keep timeout 5s every time.  it will execute 3 times in the test.
so the test will be maxed timeout 15s